sagemaker-debugger/tensorflow_profiling/callback_bottleneck.ipynb

```diff
@@ -1,8 +1,28 @@
 {
```

Comment:
<p>Can you run it successfully so it doesn't end on the ClientError?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2645/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>